services/places_response.go

```diff
@@ -0,0 +1,50 @@
+package services
+
+type PlacesResponse struct {
+	HTMLAttributions []interface{} `json:"html_attributions"`
+	NextPageToken    string        `json:"next_page_token"`
+	Results          []struct {
+		BusinessStatus string `json:"business_status"`
```

Comment:
You can nest structs so it would look like:

```go
type PlacesResponse struct {
  ...
  Results package.Result
  ...
}

type Results struct {
  ...attrs...
}
```

not strictly necessary at this point tho.

Reply:
Cool didn't know you could do that, we used an automatic go struct generator haha

Reply:
Maybe we can refactor this at some point because we also probably don't need to account for all the data we are never using.